Package.swift

```diff
@@ -43,6 +43,14 @@ let package = Package(
       name: "FirebaseAppCheck",
       targets: ["FirebaseAppCheck"]
     ),
+    .library(
+      name: "FirebaseFirestoreCombineSwift-Beta",
+      targets: ["FirebaseFirestoreCombineSwift"]
+    ),
+    .library(
+      name: "FirebaseFunctionsCombineSwift-Beta",
```

Comment:
"Beta" versus "Community" naming still needs to be resolved.

Reply:
> "Beta" versus "Community" naming still needs to be resolved.

Let's do this in a separate cleanup CL. (Also, I think there were still some open questions on the internal discussion).

Reply:
Sounds good. But, please comment in the meantime.  Our policy is the master branch is always releasable.